### PR TITLE
TEST - CSIP1 test case

### PR DESCRIPTION
**Pull request by koit**
Please review the testCase.xml for CSIP1 `mets/@OBJID`, issue #65. It is compliant with the updated XML template from PR #59. See also E-ARK-CSIP issue https://github.com/DILCISBoard/E-ARK-CSIP/issues/270.

### CODE REVIEW

File: corpus/mets-xml/mets/OBJID/testCase.xml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<testCase testable="TRUE">
+  <id specification="CSIP" version="2.0.0-DRAFT" requirementId="CSIP1" />
+  <references>
+    <reference requirementId="CSIP1" URL="http://earkcsip.dilcis.eu#CSIP1">
+      | ID | Name &amp; Location | Description &amp; usage | Cardinality &amp; Level |
+      | -- | --------------- | ------------------- | ------------------- |
+      | **CSIP1** | **Content Identification** `mets/@OBJID` | It is mandatory to use a content ID which is expressed with @OBJID. The value should be the same as the name or ID of the package (the name of the root folder) for the root METS document or the name and folder name for the representation. The OBJID must meet the principle of being unique at least across the repository. | **1..1** MUST |
+    </reference>
+    <reference requirementId="the-use-of-identifiers" URL="http://earkcsip.dilcis.eu/#the-use-of-identifiers">
```
Comment (PhillipTommerholt) [inline corpus/mets-xml/mets/OBJID/testCase.xml]: I am not sure the ID requirement goes for OBJID. I understand the ID requirement as only for XML-datatype "ID".

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
It is not my understanding that the OBJID should follow the ID xml data type rule.
If I am correct, then the files will have to have several amendments. If I am not correct then it seems fine

**Review by carlwilson — COMMENTED** [review]
Hi @koit I'm wondering about rule 3. I have a feeling we discussed this and thought that it might be better broken into two rules, one for the main package and the other for representations?

**Comment by koit** [comment]
> Hi @koit I'm wondering about rule 3. I have a feeling we discussed this and thought that it might be better broken into two rules, one for the main package and the other for representations?

I agree @carlwilson, it is done now.

**Review by carlwilson — APPROVED** [review]

**Review by PhillipTommerholt — APPROVED** [review]

**Pull request merged**